EE_API_automation/cico_run_EE_tests.sh

```diff
@@ -14,7 +14,8 @@ if [ -e "../jenkins-env" ]; then
 fi
 
 echo installing dependency
-sudo yum -y install python2 python2-pytest python2-requests python2-jmespath
+sudo yum -y install python2 python-pip
```

Comment:
sudo yum -y install python2 python-pip OR
sudo yum -y install python python-pip

Reply:
The automation is based on Python2.